webpack-demo/src/tasks.js

```diff
@@ -0,0 +1,30 @@
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Define the functions for adding tasks
+export const addTask = (description, tasks) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
```

Comment:
- [ ] kindly consider updating this line so as  the property index is set to the value of the new array length as a new list is added cause currently it having the value of the index not the length of the task  array 
- here is a hint : `  index: tasks.length+1`